src/tribler-gui/tribler_gui/tests/test_gui.py

```diff
@@ -38,56 +39,36 @@ def api_port():
 
 
 @pytest.fixture(scope="module")
-def window(api_port, tribler_api):
-    core_manager.START_FAKE_API = True
-    tribler_gui.defs.DEFAULT_API_PORT = api_port
+def window(api_port, tmpdir_factory):
+
+    api_key = hexlify(os.urandom(16)).encode('utf-8')
+    root_state_dir = str(tmpdir_factory.mktemp('tribler_state_dir'))
 
     app = TriblerApplication("triblerapp-guitest", sys.argv)
-    window = TriblerWindow(settings=QSettings(), api_port=api_port)  # pylint: disable=W0621
+    window = TriblerWindow(
+        QSettings(),
+        root_state_dir,
+        api_port=api_port,
+        api_key=api_key,
+        core_args=[str(RUN_TRIBLER_PY.absolute()), '--gui_test_mode'],
+    )  # pylint: disable=W0621
     app.set_activation_window(window)
     QTest.qWaitForWindowExposed(window)
 
     screenshot(window, name="tribler_loading")
-    wait_for_signal(window.core_manager.events_manager.tribler_started)
+    wait_for_signal(
+        window.core_manager.events_manager.tribler_started,
+        timeout=10,
```

Comment:
**Question** why did you use 10 as a timeout?

Reply:
That's a residue of testing different timeout values. It is the same as the default. I will remove it.